test.py

```diff
@@ -15,17 +15,21 @@ def func_exists(mod_path, func_name):
     try:
         with open(mod_path, "r") as file:
             module_content = file.read()
-    except FileNotFoundError:
+    except FileNotFoundError as e:
+        warnings.warn(f"SKIPPED: {mod_path},{func_name}: \n{e}")
```

Comment:
According to the comment in the next line of code, maybe:

```
msg = f"SKIPPED: Module file not found. \n{e}"
warnings.warn(msg)
```

Reply:
@seanlaw 
This and some other comments are outdated since this part is now moved to a different file. If you do not mind, I can resolve it here for now, and mention it later. Not a major concern IMO.

Reply:
Sounds good. I saw the comment but also saw that it was outdated so I ignored it